cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

```diff
@@ -7,7 +7,8 @@ To assign permissions, you need access to Access Management and the Organization
 
 * To set up and manage a private space, you need the CloudHub Network Administrator permission.
 * To view a private space, you need the CloudHub Network Viewer permission.
-* To deploy applications from Exchange, you need the Exchange Contributor and Exchange Viewer permissions.
+** To view the schedules in applications deployed to CloudHub 2.0 and Runtime Fabric, you need to assign the Exchange Viewer permission to the Business Organization that owns the asset.
```

Comment:
When the user "needs" to do something, we use "must". I get this comment a lot! hahahaha

Reply:
Yes, you’re righ! Me too! Hahaha :)